src/dnaio/_core.pyx

```diff
@@ -10,6 +10,7 @@ cdef extern from *:
     unsigned char * PyUnicode_1BYTE_DATA(object o)
     int PyUnicode_KIND(object o)
     int PyUnicode_1BYTE_KIND
+    object PyUnicode_New(Py_ssize_t size, Py_UCS4 maxchar)
```

Comment:
PyUnicode_New will be in Cython from version 3.0 onwards. (Unfortunately cython still supports python2, so it is not there yet). So we can move this block to the Cython imports when the time comes.

Reply:
Good to know. I’m also looking forward to Cython 3.